microk8s-resources/actions/disable.metrics-server.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+set -e
+
+source $SNAP/actions/common/utils.sh
+
+echo "Disabling Metrics-Server"
+
+KUBECTL="$SNAP/kubectl --kubeconfig=${SNAP_DATA}/credentials/client.config"
+# Clean up old metrics-server  
+$KUBECTL delete configmap -n kube-system metrics-server-config || true
+$KUBECTL delete deployment -n kube-system metrics-server-v0.2.1 || true 
+
+use_manifest metrics-server delete 
+# wait for pod to terminate before restarting kubelet
+echo "Waiting for pods to be terminated."
+sleep 10
+
+skip_opt_in_config "authentication-token-webhook" kubelet
+skip_opt_in_config "authorization-mode" kubelet
+
+restart_service kubelet
+kubelet=$(wait_for_service kubelet)
```

Comment:
I do not think we need to wait.

Reply:
At first i let it restart, but soon i noticed that the metrics server Pod is stuck in `Terminating` phase for quite some time.
Let me test this again.

Reply:
If there is no need to change the kubelet options, this can go away.